src/code/z_actor.c

```diff
@@ -62,9 +62,9 @@ void ActorShadow_DrawCircle(Actor* actor, Lights* lights, GlobalContext* globalC
 }
 
 void ActorShadow_DrawWhiteCircle(Actor* actor, Lights* lights, GlobalContext* globalCtx) {
-    static Color_RGBA8 white = { 255, 255, 255, 255 };
+    static Color_RGBA8 whiteCol = { 255, 255, 255, 255 };
```

Comment:
`whileColor` would be much better than `whiteCol` imo, tho `white` seems good enough to describe it tbh

Reply:
yea I already do this in many other places. i dont think specifying a color is necessary when context makes it very clear, and the type

Reply:
I felt a variable named "white" just seemed slightly weird lol, bit I can change it back or do whiteColor